jupyter-docker/Dockerfile

```diff
@@ -129,6 +127,27 @@ RUN apt-get install -yq --no-install-recommends \
  && pip install matplotlib-venn \
  && pip install "https://github.com/all-of-us/workbench/archive/pyclient-v0-1-rc3.zip#egg=aou_workbench_client&subdirectory=client/py"
 
+#######################
+# R Kernel
+#######################
+
+RUN apt-get -t jessie-backports install -y --no-install-recommends \
```

Comment:
It'd be good to do all these in the same layer (`RUN` command)`. E.g. similar to how python does the apt-gets and pips separated by &&.